lib/model.js

```diff
@@ -1885,6 +1885,35 @@ class Model {
     return this.findOne(options);
   }
 
+  /**
+ * Search for a many instances by its primary key._
+ *
+ * @param  {Array}      param The value of the desired instance's primary key.
+ *
+ * @see
+ * {@link Model.findAll}           for a full explanation of options, Note that options.where is not supported.
+ *
+ * @returns {Promise<Model>}
+ */
+  static findByPks(param) {
+  // return Promise resolved with null if no arguments are passed
+    if ([null, undefined].includes(param)) {
+      return Promise.resolve(null);
+    }
+
+    if (typeof param === 'number' || typeof param === 'number') {
+      throw new Error('The argument passed to findByPks must be an array of ids'); 
+    }
+
+    if (param.length >= 2 && Array.isArray(param) || Array.isArray(param) && param.length === 0) {
+      return this.findAll({ where: { id: param } });
+    } if (param.length == 1) {
```

Comment:
I don't see how this is useful, if I want to use `findByPks([1])` I should be able to. IMHO.